Packs/LogPoint_SIEM_Integration/Integrations/LogPoint_SIEM_Integration/LogPoint_SIEM_Integration.py

```diff
@@ -729,10 +730,15 @@ def get_searchid_command(client, args):
 
 def search_logs_command(client, args):
     search_id = args.get('search_id')
-    search_result = client.get_search_results(search_id)
-    if not search_result.get('success'):
-        raise DemistoException(search_result.get('message'))
-    rows = search_result.get('rows', [])
+    rows = []
+    while True:
+        search_result = client.get_search_results(search_id)
+        if not search_result.get('success'):
+            raise DemistoException(search_result.get('message'))
+        rows += search_result.get('rows', [])
+        if search_result.get('final'):
+            break
+        time.sleep(2)
```

Comment:
Using **time.sleep()** can waste computing resources. What is the reason for using it?

Reply:
This avoids overloading the api call. We wait 2s between each loop to avoid a crash.

Reply:
We don't accept **sleep** commands, as it can cause issues in the system. You can implement [polling](https://xsoar.pan.dev/docs/integrations/scheduled-commands) if necessary.

Reply:
Youssef, let's remove it then, I've just carried out successful local tests. If the crash problem appears later despite the timeout, I'll do a feature with polling.

Reply:
Then please update the code to not use the **sleep** function so we can conduct a small demo and proceed with this PR.

Reply:
Anas, can you do the update yourself? Or show me how to do it. Thank you